test/functional/tests/ci/test_basic.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+
+from core.test_run import TestRun
+from api.cas.version import CasVersion
+from test_utils.filesystem.file import File
+
+
+def test_cas_version():
+    """
+        title: Check if CAS is installed
+        description: Check if CAS is installed with `--version` flag
+        and later components version is compared with version file
+        pass_criteria:
+          - `casadm` command succeeds
+          - Versions are matched from cmd and file in /var/lib/opencas/cas_version
+    """
```

Comment:
Please align with Test description format as in other tests.
Also leave --version and casadm without " ` " and pass_criteria without "_".

Reply:
Done :)